torch/_dynamo/variables/builder.py

```diff
@@ -496,6 +496,17 @@ def index_source(key):
                 ),
                 "apply",
             )
+        elif isinstance(value, types.MethodType):
+            # don't let methodtypes fall through to UserDefinedObject,
+            # which doesn't support'CALL_FUNCTION'
+            return UserMethodVariable(
+                value.__func__,
+                VariableBuilder(self.tx, source=AttrSource(self.source, "__self__"))(
+                    value.__self__
+                ),
```

Comment:
If this is `None` after re-entering the builder here, will that silently pass? Consider pulling this out, adding a comment for what we do, and checking a few basic assumptions.